beta/nncf/tensorflow/sparsity/utils.py

```diff
@@ -64,22 +64,33 @@ def strip_model_from_masks(model: tf.keras.Model) -> tf.keras.Model:
     for layer in model.layers:
         if isinstance(layer, NNCFWrapper):
             for weight_attr, ops in layer.weights_attr_ops.items():
-                # BinaryMask operation must be the first operation
-                op_name, op = next(iter(ops.items()))
-                if isinstance(op, BinaryMask):
-                    apply_mask(layer, weight_attr, op_name)
-
-                    transformations.register(
-                        TFRemovalCommand(
-                            target_point=TFOperationWithWeights(
-                                layer.name,
-                                weights_attr_name=weight_attr,
-                                operation_name=op_name)
-                        ))
+                for op_name in ops.keys():
```

Comment:
Can you please refactor this according to the style guide?

Reply:
Done